File: Source/API/DTO/TurnoverSheetSummaryDTO.cs
namespace OCBManager.API.DTO
{
    public record TurnoverSheetSummaryDTO(
        int Id, string Name,
        decimal IncomingBalanceActive,
        decimal IncomingBalancePassive,
        decimal TurnoverDebit,
        decimal TurnoverCredit,
        decimal OutgoingBalanceActive,
        decimal OutgoingBalancePassive);
}

File: Source/Frontend/src/app/core/dto/turnover-sheet-summary-dto.ts
export interface TurnoverSheetSummaryDto {
    id: number, 
    name: string,
    incomingBalanceActive: number,
    incomingBalancePassive: number,
    turnoverDebit: number,
    turnoverCredit: number,
    outgoingBalanceActive: number,
    outgoingBalancePassive: number
}